src/components/GlobalStyles.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { Global, css } from "@emotion/core";
+
+const waterprimarycolor = "#032973";
```

Comment:
These variables are unused.
If you like to add a theme, take a look at https://emotion.sh/docs/theming.
This allows you to read from `props => props.theme` inside your styled-components.
The `ThemeProvider` should be added to your `App.js`. 

Without a theme, you could use CSS vars: https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties

Reply:
Thank you for the links and suggested solutions.